src/commontypes.h

```diff
@@ -14,6 +14,7 @@
 
 #include <array>
 #include <assert.h>
+#include <cstring>
```

Comment:
What does this do?

Reply:
False positive - I thought this change was fixing a build error on Ubuntu, but it still builds after removing it. It's now removed